modules/redux/test/integration.test.ts

```diff
@@ -242,4 +242,46 @@ describe("Integration", () => {
     const { entities } = store.getState().phenyl;
     assert.deepStrictEqual(entities, {});
   });
+
+  it("should be check for diff Origin and Head", () => {
```

Comment:
Better expression for this wil be `should check diff between origin and head in EntityInfo`.

Reply:
Thank you. I applied the suggested expression.